Makefile

```diff
@@ -652,63 +670,17 @@ wizard-gen:
 	gravity ops create-wizard --ops-url=$(LOCAL_OPS_URL) gravitational.io/telekube:0.0.0+latest /tmp/telekube
 
 #
-# robotest-installer builds an installer tarball for use in robotest
-# Resulting installer URL is written to a properies file specified with BUILDPROPS
-#
-.PHONY: robotest-installer
-robotest-installer: TMPDIR := $(shell mktemp -d)
-robotest-installer: BUILDPROPS ?= build.properties
-robotest-installer: LOCAL_STATE_DIR := $(TMPDIR)/state
-robotest-installer: EXPORT_DIR := $(LOCAL_STATE_DIR)/export
-robotest-installer: EXPORT_APP_TARBALL := $(EXPORT_DIR)/app.tar.gz
-robotest-installer: ROBOTEST_APP_PACKAGE ?= $(TELEKUBE_APP_PKG)
-robotest-installer: ROBOTEST_APP_PACKAGE_SRCDIR ?= $(TOP)/assets/telekube
-robotest-installer: ROBO_BUCKET_URL = s3://builds.gravitational.io/robotest
-robotest-installer: ROBO_GRAVITY_BUCKET := $(ROBO_BUCKET_URL)/gravity/$(GRAVITY_VERSION)
-robotest-installer: INSTALLER_FILE := $(subst /,-,$(subst :,-,$(ROBOTEST_APP_PACKAGE)))-$(GRAVITY_VERSION)-installer.tar.gz
-robotest-installer: INSTALLER_URL := $(ROBO_BUCKET_URL)/$(INSTALLER_FILE)
-robotest-installer: robotest-publish-gravity
-robotest-installer:
```

Comment:
Is this no longer used? How will robotest use the new application for install scenarios? IIRC currently it builds it locally?

Reply:
This is unused for many years.  More context in the commit message:

https://github.com/gravitational/gravity/pull/1967/commits/dd357c87cab1303eb537ea4c175546df2995cf56

The `robotest-image` target produces the new installer tarball which will be used for all scenarios.  I don't have a strong opinion on the name.